.ci/travis-dependencies.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
 
+if [[ $TRAVIS_OS_NAME == "linux" ]] ; then
```

Comment:
I see no reason to move this around

Reply:
It doesn't matter code wise, yes.

The idea is to have all travis scripts (travis-checks, -dependencies, -compile) with the same structure.
1. Check for `linux` (also first in travis build matrix)
2. Check for `osx`
3. Check for `Debug`
4. Check for `Release`

And don't have mac checks at the beginning of one file and at the end of the next.

Reply:
👍 consistency